src/client/components/timePeriodPicker/index.js

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+
+import TimePeriodPicker from './TimePeriodPicker';
+import { updateChartTimePeriod } from '../../actions';
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
this is super nit-picky but I'd put `mapStateToProps` above `mapDispatchToProps` in the file